packages/auto-approve/test/auto-approve.test.ts

```diff
@@ -123,6 +130,7 @@ describe('auto-approve', () => {
     checkAutoApproveStub.restore();
     checkCodeOwnersStub.restore();
     getSecretStub.restore();
+    checkPRAgainstConfigV2Stub.restore();
```

Comment:
This is much simplified if you use a sinon.sandbox and restore the entire sandbox.